### PR TITLE
feat: add points to assessment criteria in submission fetch

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** add `points` to assessment criteria api data object

JIRA: [AU-410](https://openedx.atlassian.net/browse/AU-410)

**What changed?**
- Add a field `points` to the submission fetch endpoint, within the criteria object within the assessment object
- This field represents the number of points that the chosen option for this criterion is worth (whose name is specified by `option` in the same object)

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/staffgrader/serializers/assessments.py [hunk 1]
```diff
@@ -19,10 +19,11 @@ class SubmissionDetailFileSerilaizer(serializers.Serializer):
 class AssessmentPartSerializer(serializers.ModelSerializer):
     name = serializers.CharField(source='criterion.name')
     option = serializers.CharField(source='option.name', default=None)
+    points = serializers.IntegerField(source='option.points', default=None)
```
Comment (nsprenkle) [inline openassessment/staffgrader/serializers/assessments.py:22]: Is it possible to _not_ have a `points` field? Is the default wanted/necessary?

**Review by nsprenkle — APPROVED** [review]
LGTM 👍

File: openassessment/staffgrader/serializers/assessments.py (hunk 1 above)
Comment (jansenk) [reply openassessment/staffgrader/serializers/assessments.py:22]: Yes, if we have a feedback-only criterion, this will be None
